### PR TITLE
Adds Mesh Tutorial

### PR DESCRIPTION
**Pull request by francocipollone**
Signed-off-by: Franco Cipollone <franco.c@ekumenlabs.com>

# 🎉 New feature

The mesh tutorial [here created](https://docs.google.com/document/d/18rf5p6OTQ_Fpp-cHAFvcCcETtJG7N9FPcHoqfOEAxjU/edit?usp=sharing) is migrated from Google Doc to an actual tutorial in `ign-gazebo`
**Tutorial Author**: @RafiAbdullah

Some pics:
image

image

## Test it

I share here the generated HTML  in case you want to take a look:

[model_and_optimize_meshes_tutorial.zip](https://github.com/ignitionrobotics/ign-gazebo/files/6812464/model_and_optimize_meshes_tutorial.zip)

After extracting the files you can use your favorite browser by doing: (e.g. using google chrome)
```
google-chrome model_and_optimize_meshes.html
``` 

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: tutorials/model_and_optimize_meshes.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+3d shapes created for simulation serve the purpose of bringing realism to a scene. It also helps identify everything in the scene easier. If for example all of the objects in a scene were simple shapes like cubes and circles it can be difficult to distinguish objects in the simulation.
+
+### Table of Contents
+
+ - [Collision shapes vs Visual shapes](#collision-shapes-vs-visual-shapes)
+ - [Creating Models](#creating-models)
+   - [Create/add simple mesh](#create/add-simple-mesh)
+   - [More polygons less performance](#more-polygons-less-performance)
+ - [Exporting Models](#exporting-models)
+ - [Troubleshooting](#troubleshooting)
```
Comment (chapulina) [inline tutorials/model_and_optimize_meshes.md:12]: These anchors don't work with Doxygen, unfortunately

File: tutorials/model_and_optimize_meshes.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+When you export a model for Gazebo, the .dae (COLLADA) file type is recommended.
+STL and OBJ files are also supported, but are less common for imports into Gazebo.
+
+To export: select your shape and then `File>Export>Collada(Default)(.dae)`
+
+@image html files/model_and_optimize_meshes/14_export_model.png width=600px
+
+After you hit export in whatever file format you decide to go with, the "Blender File View" will appear. This is where you choose the location of your file. You can also choose how you want to export your model.
+
+Double checking the “global Orientation” can save time and here you can quickly change the “Up Axis” to Y up. This is the standard orientation in Gazebo
```
Comment (chapulina) [inline tutorials/model_and_optimize_meshes.md:109]: Gazebo uses Z up, I'm checking with @RafiAbdullah

**Review by chapulina — COMMENTED** [review]
I just have one technical comment about the anchors and a question that I'm checking with Rafi. Otherwise looks good to me :+1:

File: tutorials/model_and_optimize_meshes.md (hunk 2 above)
Comment (chapulina) [reply tutorials/model_and_optimize_meshes.md]: Confirmed that it should be Z up and updated in b121607d17c95701d6eb0531beb1884fefdce3ea

File: tutorials/model_and_optimize_meshes.md (hunk 1 above)
Comment (chapulina) [reply tutorials/model_and_optimize_meshes.md]: I just removed them for now b121607d17c95701d6eb0531beb1884fefdce3ea

**Review by chapulina — APPROVED** [review]

**Comment by francocipollone** [comment]
Thanks for pushing the changes! @chapulina

**Pull request merged**